### DISCUSSION TITLE
The Last King Of Hyrule (TheXTech Repack)

### DISCUSSION THREAD

**Discussion by DarkWeBareBears69** [Wohlstand/TheXTech#808] [Game projects]
I decided to repack the game project "The Last King of Hyrule", founded on rpgmaker.net and since I didn't play Zelda 2 back then, there are some unnecessary changes in the repack, you can change them by pulling the request.

[Source code](https://github.com/DarkWeBareBears69/The-Last-King-Of-Hyrule-TheXTech-Repack)

**Comment by Wohlstand** [comment]
Hello!

I tried this thing on my end, and it looks very interesting. First in my mind that you should replace all the MP3 music with NSF / SPC files that will sound clear, will loop correctly, and will take very few space (you can actually adjust their volume by `g=x.yy;` arguments). Here is complete manual how to use MixerX's music arguments: https://wohlsoft.ru/pgewiki/SDL_Mixer_X#Path_arguments

Additionally, a small note: if some levels supposed as a place of relaxing like village or castle, you can mark them as "sub-hubs", and player will be able to save on them and resume at them instead of the world map. However, to enable this feature, you should convert the whole episode into TheXTech format (LVLX+WLDX formats) using latest Moondust Maintainer from the Laboratory builds.
